libcodechecker/util.py

```diff
@@ -266,7 +266,7 @@ def split_server_url(url):
     LOG.debug("Parsing server url '{0}'".format(url))
 
     protocol = 'http'
-    if url.startswith('http'):
+    if url.startswith('http') or url.startswith('https'):
```

Comment:
If url starts with `https` it also starts with `http` right?  This check might be redundant.